tests/mypy/test_plugin.yml

```diff
@@ -36,3 +36,16 @@
     NameInterface(name="Patrick")
   out: |
     main:7: error: Unexpected keyword argument "n" for "NameInterface"
+
+- case: test_field
+  main: |
+    import strawberry
+
+    @strawberry.type
+    class User:
+        name: str = strawberry.field(description='Example')
+
+    User(n="Patrick")
+    User(name="Patrick")
```

Comment:
Does this line ever get hit?

Reply:
yes :) I've inverted the lines so it is more clear that it runs that line too :)